--- dvtm-0.5.1-orig/Makefile	2009-02-09 22:00:22.128338075 +0100
+++ dvtm-0.5.1/Makefile	2009-02-09 22:04:50.326161374 +0100
@@ -5,15 +5,18 @@
 
 all: clean options dvtm
 
+OPT_FLAGS ?= -Os
+
 options:
 	@echo dvtm build options:
 	@echo "CFLAGS   = ${CFLAGS}"
+	@echo "OPT_FLAGS= ${OPT_FLAGS}"
 	@echo "LDFLAGS  = ${LDFLAGS}"
 	@echo "CC       = ${CC}"
 
 .c.o:
 	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	@${CC} -c ${CFLAGS} ${OPT_FLAGS} $<
 
 ${OBJ}: config.h config.mk
 
--- dvtm-0.5.1-orig/config.mk	2009-02-09 22:00:22.128338075 +0100
+++ dvtm-0.5.1/config.mk	2009-02-09 22:01:50.141803270 +0100
@@ -10,7 +10,7 @@
 LIBS = -lc -lutil -lncursesw
 #LIBS = -lc -lutil -lncurses
 
-CFLAGS += -std=c99 -Os ${INCS} -DVERSION=\"${VERSION}\" -DNDEBUG
+CFLAGS += -std=c99 ${INCS} -DVERSION=\"${VERSION}\" -DNDEBUG
 LDFLAGS += -L/usr/lib -L/usr/local/lib ${LIBS}
 
 # Mouse handling
